src/auto-upgrade/updatePackage.js

```diff
@@ -12,6 +12,23 @@ const PROJECT_PACKAGE_LOCATION = path.join(process.cwd(), "package.json");
 // don't have to do file I/O more than once
 let _projectPackageData;
 
+
+/**
+ * This function is used to put the dependencies in alphabetical order.
+ * It takes a hash and returns the same hash ordered by its keys.
+ *
+ * @param {Object} Unordered hash which one would like to be sorted by keys
+ *
+ * @return {Object}
+ */
+function orderHashByKeys (unordered) {
+  const ordered = {};
```

Comment:
So you can't stringify Map objects… I looked into what `npm` uses and they use an module called `sorted-object` https://www.npmjs.com/package/sorted-object

It works really well.

Try this sample code in the following url:
https://tonicdev.com/npm/sorted-object
```
var sO = require("sorted-object");
var x = {
  a: "Todd",
  z: "Zeke",
  b: "Ambre"
};
var y = sO(x);
JSON.stringify(y);
```